Makefile

```diff
@@ -129,7 +129,7 @@ endif #CONFIG
 
 BASE_CONFIGS      = $(sort $(notdir $(patsubst %/,%,$(dir $(wildcard $(ROOT)/src/config/*/config.h)))))
 BASE_TARGETS      = $(sort $(notdir $(patsubst %/,%,$(dir $(wildcard $(ROOT)/src/main/target/*/target.mk)))))
-CI_TARGETS       := $(BASE_TARGETS)
+CI_TARGETS       := $(BASE_TARGETS) CRAZYBEEF4SX1280 CRAZYBEEF4FR IFLIGHT_BLITZ_F722
```

Comment:
Why do we need to add this targets?

Reply:
We don't but I've added a few of the most popular so we can check they also build ok

Reply:
Can we also please add:

1) SPRacingH7EXTREME to that list because it's an the only EXST H750 target that uses quad SPI and there are no other targets that check a) the EXST build system for H750 and b) that compile QuadSPI.

2) the SPRacingH7RF as it's the only EXST H730 target that uses octospi and memory mapped code as there are no other targets that check a) the EXST build system for H730 and b) that compile the OctoSPI and CONFIG_IN_MEMORY_MAPPED_FLASH code

Currently there is no CI visibity on any of the above and build or compilation issues are likely to creep in un-noticed.